x-pack/test/security_solution_cypress/cypress/objects/timeline.ts

```diff
@@ -131,6 +131,7 @@ export const expectedExportedTimeline = (timelineResponse: Cypress.Response<Time
       { id: 'event.category' },
       { id: 'event.action' },
       { id: 'host.name' },
+      { id: 'message' },
```

Comment:
Did some tests expect the message column to be populated?

Reply:
Yes!